package.json

```
@@ -5,7 +5,7 @@
     "publishConfig": {
         "access": "public"
     },
-    "version": "0.0.3",
+    "version": "0.0.5",
```

```suggestion
    "version": "0.0.7",
```

seems we have 0.0.6 on npmjs already